explainer.md

```diff
@@ -1,133 +1,156 @@
-  }
-};
+[The push API](https://w3c.github.io/push-api/) allows the server to dictate when the service worker should wake up and seek updates, but these are not sensitive to connection and charging state. Also, some sites update too frequently to warrant a push message per update (think Twitter, or a news site).
+
+Periodic syncs are simple to set up, don't require any server configuration, and allow the UA to optimize when they fire to be most-helpful and least-disruptive to the user. E.g. if the UA knows the user has a morning alarm set, it may run syncronizations shortly beforehand, giving the user quick and up-to-date information from their favourite sites.
+
+### The API
+
+**To request a periodic sync:**
+
+```js
+navigator.serviceWorker.ready(function(registration) {
+  registration.periodicSync.register({
+    tag: 'get-latest-news',         // default: ''
+    minPeriod: 12 * 60 * 60 * 1000, // default: 0
+    powerState: 'avoid-draining',   // default: 'auto'
+    networkState: 'avoid-cellular'  // default: 'online'
+  }).then(function(periodicSyncReg) {
+    // success
+  }, function() {
+    // failure
+  })
+});
 ```
-The `waitUntil` is a signal to the UA that the sync event is ongoing and that it should keep the SW alive if possible. Rejection of the event signals to the UA that the sync failed. Upon rejection the UA should reschedule (likely with a UA-determined backoff).
 
-## Removing Sync Events
-If the id is not registered the function will reject.
+* `tag`: This operates like a notification's tag. If you register a sync and an existing sync with the same tag is pending, it returns the existing registration. If further options are added to `.register`, the existing registration will be updated. **Note:** this tag is independent of one-off sync tags.
+* `minPeriod`: The minimum time between successful sync events. A value of 0 (the default) means the UI may fire the event has frequently as it wishes. This value is a suggestion to prevent over-syncing. Syncing may be less frequent depending on heuristics such as visit frequency & device status. If timing is critical, use push messaging.
+* `powerState`: Either "auto" (default) or "avoid-draining". "avoid-draining" will delay syncs on battery-powered devices while that battery isn't charging. "auto" allows syncs to occur during battery-drain, although the UA may choose to avoid this depending on global device status (such as battery-saving mode) or user preferences.
+* `networkState`: One of "online" (default), "avoid-cellular", or "any". "avoid-cellular" will delay syncs if the device is on a [cellular connection](https://w3c.github.io/netinfo/#idl-def-ConnectionType.cellular). "online" will delay syncs if the device is online, although the UA may choose to avoid particular connection types depending on global device status (such as roaming) or user preferences. "any" is similar to "online", except syncs may happen while the device is offline.
```

Comment:
`avoid-cellular` is potentially a foot-gun. Many users never connect to wifi on their devices. We still need to have the option, developers just need to be careful with it.